### DISCUSSION TITLE
Calendar Questions

### DISCUSSION THREAD

**Discussion by MexasPT** [YetiForceCompany/YetiForceCRM#14668] [How to]
Hello everyone!
I am having a hard time understanding two issues. For the past two weeks I have been trying to solve it myself but without success.

**Question 1:**
As you can see in image 1, there is an icon for "send e-mail notification about Event".
In my installation I don't have the icon, probably because I removed it accidentally. Can you tell me what I need to do to get the icon back?

**Question 2:**
What do I need to do to send the email to the external user?
What does "Awaiting acceptance" mean?

**Note:**
-- Cron is working correctly;
-- SMTP is working correctly;
-- The configuration of the roundcube is Ok and sends email successfully; 

![Captura Web_14-5-2021_111916_gitstable yetiforce](https://user-images.githubusercontent.com/17734465/118258076-5693d880-b4a7-11eb-8f79-82faf56a4101.jpg)

**Comment by MexasPT** [comment]
Please help me...

**Comment by MexasPT** [comment]
Please help me :(

**Reply by rskrzypczak** [reply]
`Question 1:`
Do you have _Internal mail composer_ active in user configuration?

**Reply by MexasPT** [reply]
Hi rskrzypczak!
Thank you for your help.
Yes, i have.

**Comment by MexasPT** [comment]
What does "Awaiting acceptance" mean?
How do i change the status?

Help me please.

**Comment by MexasPT** [comment]
I just want to understand the process...
Can anyone explain it to me?

Thanks

**Reply by apcloic** [reply]
@MexasPT 

Hi, regarding question 2, if everything is properly configured on your system, you only have to click on "+" in edit view to open the popup.

![calendar_invite_external_user](https://user-images.githubusercontent.com/2930560/123834682-fa2a3280-d907-11eb-84c1-72562182e5c1.gif)

Regards,

**Comment by MexasPT** [comment]
That i understand...
But... What does "Awaiting acceptance" mean?
And how do i change it?

**Reply by mariuszkrzaczkowski** [reply]
this mechanism has not been completed, this means the presence status of the invitee

**Comment by antcorro** [comment]
Hello,

I have the same problem. In my YetiForce v6.3.0 SecurityFix v4 I don't have the icon 1 (image above) and this is just what I need.
Any help/suggestion? What are the requirements to show the icon?

Thanks

**Comment by mariuszkrzaczkowski** [comment]
here are the conditions for displaying this button
https://github.com/YetiForceCompany/YetiForceCRM/blob/c56ff382f5d611bd00d42362d220dbbbe0f1bf40/modules/Calendar/models/DetailView.php#L31

**Reply by antcorro** [reply]
ok thank you. Now I understand. I need to enable also "My mailbox" module, becausa the invitation is send from my mailbox.

I think in my case it's better to setup a workflow to send invitation from default SMTP when the record is created.

Thank you.
